spec/basic-concepts.md

```diff
@@ -781,93 +799,88 @@ A *type-name* is a *namespace-or-type-name* that refers to a type. Following res
 
 The meaning of a *namespace-or-type-name* is determined as follows:
 
 -   If the *namespace-or-type-name* is a *qualified-alias-member*, the meaning is as specified in §14.8.1.
 
--   Otherwise, if the *namespace-or-type-name* is of the form I or of the form I&lt;A~1~, …, A~K~&gt;:
+-   Otherwise, if the *namespace-or-type-name* is of the form `I` or of the form `I<A1, …, AK>`:
 
 <!-- -->
 
--   If K is zero and the *namespace-or-type-name* appears within a generic method declaration (§15.6) but outside the *attributes* of its *method-header,* and if that declaration includes a type parameter (§15.2.3) with name I, then the *namespace-or-type-name* refers to that type parameter.
+-   If `K` is zero and the *namespace-or-type-name* appears within a generic method declaration (§15.6) but outside the *attributes* of its *method-header,* and if that declaration includes a type parameter (§15.2.3) with name `I`, then the *namespace-or-type-name* refers to that type parameter.
 
--   Otherwise, if the *namespace-or-type-name* appears within a type declaration, then for each instance type T (§15.3.2), starting with the instance type of that type declaration and continuing with the instance type of each enclosing class or struct declaration (if any):
+-   Otherwise, if the *namespace-or-type-name* appears within a type declaration, then for each instance type `T` (§15.3.2), starting with the instance type of that type declaration and continuing with the instance type of each enclosing class or struct declaration (if any):
 
 <!-- -->
 
--   If K is zero and the declaration of T includes a type parameter with name I, then the *namespace-or-type-name* refers to that type parameter.
+-   If `K` is zero and the declaration of `T` includes a type parameter with name `I`, then the *namespace-or-type-name* refers to that type parameter.
 
--   Otherwise, if the *namespace-or-type-name* appears within the body of the type declaration, and T or any of its base types contain a nested accessible type having name I and K type parameters, then the *namespace-or-type-name* refers to that type constructed with the given type arguments. If there is more than one such type, the type declared within the more derived type is selected. \[*Note*: Non-type members (constants, fields, methods, properties, indexers, operators, instance constructors, finalizers, and static constructors) and type members with a different number of type parameters are ignored when determining the meaning of the *namespace-or-type-name*. *end note*\]
+-   Otherwise, if the *namespace-or-type-name* appears within the body of the type declaration, and `T` or any of its base types contain a nested accessible type having name I and `K` type parameters, then the *namespace-or-type-name* refers to that type constructed with the given type arguments. If there is more than one such type, the type declared within the more derived type is selected. 
```

Comment:
a missing code-fence:

```suggestion
-   Otherwise, if the *namespace-or-type-name* appears within the body of the type declaration, and `T` or any of its base types contain a nested accessible type having name `I` and `K` type parameters, then the *namespace-or-type-name* refers to that type constructed with the given type arguments. If there is more than one such type, the type declared within the more derived type is selected. 
```